Documentation/source/Getting started.md

```diff
@@ -45,3 +46,115 @@ let documentService: AccountingDocumentService = giniPayApiLib.documentService()
+
+The Gini Pay API lib supports creating documents from images, PDFs or UTF-8 encoded text. Images are usually a picture of a paper document which was taken with the device’s camera.
+
+The following example shows how to create a new document from a byte array containing a JPEG image.
+```swift
+documentService.createDocument(fileName: "myFirstDocument.jpg",
+                            docType: docType,
+                            type: .partial(document.data),
+                            metadata: metadata) { result in
+    switch result {
+    case let .success(createdDocument):
+        print("📄 Created document with id: \(createdDocument.id) " +
+            "for vision document \(document.id)")
+        completion(.success(createdDocument))
+    case let .failure(error):
+        print("❌ Document creation failed: \(error)")
+        completion(.failure(error))
+    }
+}
+```
+
+Each page of a document needs to uploaded as a partial document. In addition documents consisting of one page also should be uploaded as a partial document.
+
+> ⚠️  **Warning**
+> - If you are using the Gini Accounting API, then partial documents are not supported. Use the `documentService.createDocument()` methods instead.
+
+> ⚠️  **Note**
+> - PDFs and UTF-8 encoded text should also be uploaded as partial documents. Even though PDFs might contain multiple pages and text is “pageless”, creating partial documents for these keeps your interaction with Gini consistent for all the supported document types.
+
+Extractions are not available for partial documents. Creating a partial document is analogous to an upload. For retrieving extractions see Getting extractions section.
+
+> ⚠️  **Note**
+> - The `filename` (myFirstDocument.jpg in the example) is not required, it could be nil, but setting a filename is a good practice for human readable document identification.
+
+#### Setting the document type hint
+
+To easily set the document type hint we introduced the `DocType` enum. It is safer and easier to use than a String. For more details about the document type hints see the Document Type Hints in the [Gini Pay API documentation](https://pay-api.gini.net/documentation/#document-types)
+
+### Getting extractions
+
+After you have successfully created the partial documents, you most likely want to get the extractions for the document. Composite documents consist of previously created partial documents. You can consider creating partial documents analogous to uploading pages of a document and creating a composite document analogous to processing those pages as a single document.
+
+> ⚠️  **Warning**
+> - If you are using the Gini Accounting API, then composite documents are not supported. You need to use `documentService.fetchDocument()` and then `documentService.extractions()` without creating a composite document.
```

Comment:
Also not needed anymore. The accounting api works with composites.

Reply:
removed 👍